.github/lock.yml

```diff
@@ -10,7 +10,7 @@ skipCreatedBefore: false
 # Issues and pull requests with these labels will be ignored. Set to `[]` to disable
 exemptLabels:
    - "STATE: Need response"
-   - "STATE: Need clarification"
+   - "STATE: Waiting for customer to reply"
```

Comment:
may be 
'Awaiting customer response' or ' Waiting for customer'

Reply:
@RomanTsukanov what do you think about these changes?

Reply:
"Awaiting customer response" is a good alternative